MSAL/src/MSALPublicClientApplication+Internal.h

```diff
@@ -27,8 +27,8 @@
 
 #import "MSALPublicClientApplication.h"
 
-// TODO: Remove these defaults once uid/client info is in prod
-#define DEFAULT_SLICE_PARAMS @"slice" : @"testslice", @"uid" : @"true"
+// If modifying this make sure UT_SLICE_PARAMS_QUERY matches
```

Comment:
Hm do we still need the defaultSliceParameters function then? Maybe get rid of all of that? Or are we expecting devs to be modifying this? If so, in what scenario, and what documentation exists for it?